includes/REST_API/Link_Controller.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/**
+ * Class Link_Controller
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\REST_API;
+
+use WP_Error;
+use WP_REST_Controller;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * API endpoint to allow parsing of metadata from a URL
+ *
+ * Class Link_Controller
+ */
+class Link_Controller extends WP_REST_Controller
+{
+
+    /**
+     * Constructor.
+     */
+    public function __construct()
+    {
+        $this->namespace = 'amp/v1';
```

Comment:
We should totally use the `web-stories` prefix for all our REST API endpoints.

Bit of an oversight.

Reply:
Handled in #346.